com.unity.render-pipelines.high-definition/Runtime/Material/Lit/Lit.hlsl

```diff
@@ -2014,7 +2014,7 @@ IndirectLighting EvaluateBSDF_Env(  LightLoopContext lightLoopContext,
         lighting.specularReflected = envLighting;
 #if HAS_REFRACTION
     else
-        lighting.specularTransmitted = envLighting * preLightData.transparentTransmittance;
+        lighting.specularTransmitted = envLighting;
```

Comment:
can you check we don't have the same issue in stacklit (iirc stacklit support refraction?)

Reply:
Nope, looks like it is not supported